selfdrive/car/ford/carstate.py

```diff
@@ -106,6 +109,7 @@ def get_can_parser(CP):
       ("AccStopMde_D_Rq", "EngBrakeData"),                   # PCM ACC standstill
       ("AccEnbl_B_RqDrv", "Cluster_Info1_FD1"),              # PCM ACC enable
       ("StePinComp_An_Est", "SteeringPinion_Data"),          # PSCM estimated steering angle (deg)
+      ("StePinCompAnEst_D_Qf", "SteeringPinion_Data"),       # PSCM estimated steering angle (quality flag)
```

Comment:
This signal sometimes remains at 0 while the ABS is calibrating the steering pinion offset (`BrakeSnData_6->StePinOffst_An_Est`), and so the steering angle is a static 0 -- we should not engage here.

Reply:
this should be a comment in the code